src/test/java/io/debezium/connector/vitess/AbstractVitessConnectorTest.java

```diff
@@ -472,7 +475,18 @@ else if (actualValue instanceof Struct) {
                 assertStruct((Struct) value, (Struct) actualValue);
             }
             else {
-                assertEquals("Values don't match for field '" + fieldName + "'", value, actualValue);
+                Schema schema = content.schema().field(fieldName).schema();
+                if (schema.name() != null && !schema.name().isEmpty() && schema.name().equals(Json.LOGICAL_NAME)) {
```

Comment:
Could this simply be `if(Json.LOGICAL_NAME.equals(schema.name()) {`?

Reply:
Indeed, your `if condition` is much more simplified. Thanks. Addressed.